release/NativeEditPlugin/Plugins/iOS/EditBox_iOS.m

```diff
@@ -447,7 +463,23 @@ -(void) create:(JsonObject*)json
         [textField addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
         [textField setSecureTextEntry:password];
         if (keyboardDoneButtonView != nil) textField.inputAccessoryView = keyboardDoneButtonView;
-        
+
+        if (@available(iOS 12.0, *))
```

Comment:
Can you please this verbatim duplication of the override? In case we want to add more overridable types, there's a risk only one of these will be modified. There may be other duplication between multiline and singleline, but let's not add more.

Reply:
Hmm, i guess the only way to do that atm is to cast it after it's set in the editview? It's not that clean either, but I'll try.

Reply:
Fixed., no need for casting